core/dbt/config/project.py

```diff
@@ -130,7 +135,7 @@ def package_config_from_data(packages_data):
         packages = PackageConfig.from_dict(packages_data)
     except ValidationError as e:
         raise DbtProjectError(
-            'Invalid package config: {}'.format(validator_error_message(e))
+            MALFORMED_PACKAGE_ERROR
```

Comment:
Let's retain the error message here. It is helpful, and it definitely benefits from some additional context. I think the stringified error message is a little bit easier to understand than the output of `validator_error_message()` here though

```suggestion
            MALFORMED_PACKAGE_ERROR.format(error=str(e))
```